aws/resource_aws_appautoscaling_target_test.go

```diff
@@ -85,6 +97,12 @@ func TestAccAWSAppautoScalingTarget_emrCluster(t *testing.T) {
 					resource.TestCheckResourceAttr("aws_appautoscaling_target.bar", "scalable_dimension", "elasticmapreduce:instancegroup:InstanceCount"),
 				),
 			},
+			{
+				ResourceName:      "aws_appautoscaling_target.bar",
+				ImportState:       true,
+				ImportStateIdFunc: testAccAWSAppautoscalingTargetImportStateIdFunc("aws_appautoscaling_target.bar"),
+				ImportStateVerify: true,
+			},
 		},
 	})
 }
```

Comment:
I coundn't add test step to import resource in `TestAccAWSAppautoScalingTarget_multipleTargets` in `resource_aws_appautoscaling_target_test.go`. Because `app_awsautoscaling_target` uses `resource_id` as ID,  `ImportStateVerify` mistakingly choose resources.

Reply:
Thanks for the note 👍